apps/website/src/content/docs/flex.mdx

```diff
@@ -7,14 +7,36 @@ group: utilities
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='flex--basic' />
-
 <LiveExample src='Flex.main.jsx'>
   <AllExamples.FlexMainExample client:load />
 </LiveExample>
 
 A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens. Supports all flex properties. Can be used with or without `Flex.Item` and `Flex.Spacer` subcomponents.
 
-## Props
+## Usage
+
```

Comment:
It would be helpful to have an intro paragraph under the "Usage" heading (before any other subheadings). Here, we can explain the default behavior of `Flex` as well as mention top-level props (such as `gap`).

Reply:
I have added more explanation for the "Usage" section.